Sources/Orbit/Components/ChoiceTile.swift

```diff
@@ -149,7 +149,7 @@ public struct ChoiceTile<Content: View>: View {
 
     @ViewBuilder var indicatorOverlay: some View {
         indicatorElement
-            .disabled(true)
+            .allowsHitTesting(false)
```

Comment:
What's the difference between this and the previous version?

Reply:
Have not found yet a better way for these compositions. The scenario is that we want to display the indicator (radio/checkbox) **as if** it was enabled (disabling it here would cause it to be greyed out), yet we want it to not be accessible manually (accessibility itself is solved by the component, so the indicator alone should not be **interactive** by itself by any means).